fulp_modules/lisa/velvet_fu.dm

```diff
@@ -0,0 +1,292 @@
+			return
+
+/datum/martial_art/velvetfu/proc/receding_stance(mob/living/carbon/user)
+	return // Empty return because we only want the button trigger
+
+/// Twisted Stance
+/datum/action/twisted_stance
+	name = "Twisted Stance - Regenerates a lot of stamina, deals brute damage."
+	icon_icon = 'fulp_modules/lisa/sprites/stances.dmi'
+	button_icon_state = "twisted_stance"
+
+/datum/action/twisted_stance/Trigger(mob/living/M, mob/living/user)
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't do stances while incapacitated...</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if(owner.mind.martial_art.streak == "twisted_stance")
+		owner.visible_message("<span class='danger'>[owner] untwists [owner.p_them()]self.</i></b>")
+		owner.mind.martial_art.streak = ""
+	else
+		owner.visible_message("<span class='danger'>[owner] suddenly twists and turns, what a strange stance!</span>", "<b>You twist and turn, your twisted stance is done!</b>")
+		owner.mind.martial_art.streak = "twisted_stance"
+		H.adjustStaminaLoss(-40)
+		H.apply_damage(18, BRUTE, BODY_ZONE_CHEST, wound_bonus = CANT_WOUND)
+
+/datum/martial_art/velvetfu/proc/twisted_stance(mob/living/carbon/user)
+	return // Empty return because we only want the button trigger
+
+
+/// Flying Axe Kick - Deals Brute and causes bleeding. Costs 50 Stamina.
+/datum/martial_art/velvetfu/proc/flyingAxekick(mob/living/A, mob/living/D)
+	log_combat(A, D, "flying kick (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] flying kicked [D], such skill!</span>", \
+					"<span class='userdanger'>Your neck is flying kicked by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You flying kick [D]!</span>")
+	A.adjustStaminaLoss(50)
+	if(prob(60))
+		if(!D.stat)
+			var/obj/item/bodypart/limb = D.get_bodypart(ran_zone(A.zone_selected))
+			var/datum/wound/slash/moderate/crit_wound = new
+			crit_wound.apply_wound(limb)
+	else
+		to_chat(A, "<span class='danger'>Your flying axe kick fails to cause [D] to bleed!</span>")
+	D.apply_damage(10, BRUTE) // Slash!
+	playsound(get_turf(A), 'sound/weapons/slice.ogg', 50, TRUE, -1)
+	return TRUE
+
+/// Goat Headbutt - Deals Brute and Stuns, in exchange for causing Brute to the user. Costs 60 Stamina and deals some Brute.
+/datum/martial_art/velvetfu/proc/goatHeadbutt(mob/living/A, mob/living/D)
+	log_combat(A, D, "goat headbutt (Velvet-Fu)")
+	D.visible_message("<span class='danger'>[A] headbutted [D]!</span>", \
+					"<span class='userdanger'>You're headbutted by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You swiftly headbutt [D]!</span>")
+	A.apply_damage(15, BRUTE, BODY_ZONE_HEAD, wound_bonus = CANT_WOUND)
+	A.adjustStaminaLoss(60)
+	if(prob(80))
+		if(!D.stat)
+			D.Stun(4 SECONDS)
```

Comment:
4 second hardstun 🤮

Reply:
since the last move of this combo is a grab, does it inherit the 1 second click cooldown from that grab instead of the normal 0.8 second click cooldown for most melee attacks? if not, you can just goat headbutt someone and cuff them while they're hardstunned.